addons/payment_transfer/models/payment_transaction.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+
+import logging
+import pprint
+
+from odoo import _, api, models
+from odoo.tools.float_utils import float_compare
+
+from odoo.addons.payment.models.payment_acquirer import ValidationError
+
+_logger = logging.getLogger(__name__)
+
+
+class TransferPaymentTransaction(models.Model):
```

Comment:
Nitpicking (same as above)
```suggestion
class PaymentTransaction(models.Model):
```